src/angular/core/option/optgroup.html

```diff
@@ -1,4 +1,4 @@
 <label class="sbb-label sbb-optgroup-label" [id]="_labelId"
   >{{ label }} <ng-content></ng-content
 ></label>
-<ng-content select="sbb-option, ng-container, sbb-autocomplete-hint"></ng-content>
```

Comment:
sbb-autocomplete-hint should still work for backwards compatibility

Reply:
I was saying the same, but it works as it is!

Reply:
Yes, it's not needed. However, I don't really understand why and therefore added the selector.